tests/sentry/api/endpoints/test_source_map_debug.py

```diff
@@ -389,6 +391,8 @@ def test_no_url_match(self):
         assert error["data"] == {
             "absPath": "https://app.example.com/static/static/js/main.fa8fe19f.js",
             "fileName": "/static/static/js/main.fa8fe19f.js",
+            "artifactNames": ["http://example.com/application.js"],
```

Comment:
@roggenkemper if it's purely for debugging then I wouldn't make the tests depend on it but nbd